### PR TITLE
Adds Fluff Item "Tacticool EyePro"

### PR DESCRIPTION
**Pull request by Ty-Omaha**
Tacticool level 5

Just SecHUDs but these are way more tacticool 

ef42b5050530d4b90ab8d65ed79013cd

b6ef76c65c4d10f98c654f27e4c02379

ade4d43c90a1a22722518367b4ee9614

:cl:
add: Adds fluff item "Tacticool EyePro"
/:cl:

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -279,6 +279,15 @@
 
 //////////// Eye Wear ////////////
 
+/obj/item/clothing/glasses/hud/security/fluff/eyepro //T0EPIC4U: Ty Omaha
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: These should be a child of sec-hud sunglasses, not security HUDs.

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -278,6 +278,14 @@
...
 
 //////////// Eye Wear ////////////
+/obj/item/clothing/glasses/hud/security/sunglasses/fluff/eyepro //T0EPIC4U: Ty Omaha
+	name = "Tacticool EyePro"
+	desc = "Tacticool ballistic glasses, for making all operators look badass."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "eyepro"
+	item_state = "eyepro"
+	flash_protect = 1
+	tint = 1
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: Now get rid of the above two vars.

**Comment by Fox-McCloud** [comment]
looks good now.

**Comment by KasparoVy** [comment]
Whipped up a Vox compatible version since pretty much all eyewear (namely the parent object of these glasses) are species_fitted to them

[vox eyes.zip](https://github.com/ParadiseSS13/Paradise/files/429216/vox.eyes.zip)
voxeyepro

**Comment by FlattestGuitar** [comment]
Could use some shading, @KasparoVy

**Comment by KasparoVy** [comment]
@FlattestGuitar The base sprite didn't have none and I just added/removed pixels as necessary to get it to fit Vox. I'll wait to see how the OG spriter handles that before making changes as IDK the wishes of the person who donated for this thing.

**Comment by FlattestGuitar** [comment]
@KasparoVy good point. 

@Ty-Omaha shading when.

**Comment by KasparoVy** [comment]
Whipped up a Vox compatible version since pretty much all eyewear (namely the parent object of these glasses) are species_fitted to them
Think you could drop the file from the zip into icons/mob/species/vox and replace eyes.dmi?

voxeyepro

**Sprite amended. Has some shading and lense translucency.**

**Comment by Fox-McCloud** [comment]
@Necaladun 

Will need you to confirm the donation.

**Comment by Ty-Omaha** [comment]
@FlattestGuitar http://8541tactical.com/images/ESS/crossbow_7189.jpg NO SHADING. :knife: :cry: :angry:

@KasparoVy FINE BUT NO KILLING ME AND STEALING MY GLASSES :(

**Comment by KasparoVy** [comment]
@Ty-Omaha No worries, I'm just keeping up sprite parity.

**Comment by FlattestGuitar** [comment]
:eyeglasses: :gun: 
:boom: :gun: 
:cry: :laughing:

**Comment by Fox-McCloud** [comment]
@TheDZD @Earthdivine @Necaladun this will need to be added to the fluff items table.

**Pull request merged**